doc/manual/installation.md

```diff
@@ -110,7 +110,7 @@ Create a user for Huginn do not type the `mysql>`, this is part of the prompt. C
 
 Ensure you can use the InnoDB engine which is necessary to support long indexes
 
-    mysql> SET storage_engine=INNODB;
+    mysql> SET default_storage_engine=INNODB;
```

Comment:
`default_storage_engine` appears to exist in the 5.5 docs, so I guess this is fine?